clinica/utils/input_files.py

```diff
@@ -401,24 +401,49 @@ def t1_volume_native_tpm_in_mni(tissue_number, modulation):
     }
 
 
-def t1_volume_template_tpm_in_mni(group_label, tissue_number, modulation):
+def t1_volume_template_tpm_in_mni(group_label, tissue_number, modulation, fwhm=None):
+    """Build the dictionary required by clinica_file_reader to get the tissue
+    probability maps based on group template in MNI space.
+
+    Parameters
+    ----------
+    group_label : str
+        Label used for the group of interest.
+
+    tissue_number : int
+        An integer defining the tissue of interest.
+
+    modulation : {"on", "off"}
+        Whether modulation is on or off.
+
+    fwhm : int, optional
+        The smoothing kernel in millimeters.
```

Comment:
Can FWHM be a float?

Reply:
We would not want it based on our naming convention, but I am asking if it makes sense from a domain perspective.

Reply:
I left them as integers because this is how they are represented in Clinica, but I believe they could (and should) be floats.